en/includes/guides/branding/customize-sms-templates.md

```diff
@@ -2,6 +2,24 @@
 
 The following topics explain how you can customize SMS notifications that are sent to users of your {{ product_name }} organization.
 
+{% if product_name == "Asgardeo" %}
+
+!!! note "SMS templates for B2B applications"
+
+    If you have set up [organizations]({{base_path}}/guides/organization-management/manage-organizations/),
+    these organizations automatically inherit the sms templates of the root organization.
+
+{% else %}
+
+!!! note "SMS templates for B2B applications"
+
+    If you have set up [organizations]({{base_path}}/guides/organization-management/manage-organizations/),
+    you can customize sms templates to fit the branding needs of each organization. If you do not customize a specific
+    template, the sms template will be inherited from the closest ancestor organization with defined sms template.
```

Comment:
```suggestion
    sms template for an organization, it will be inherit the design from the closest ancestor organization with a customized sms template.
```